addons/udes_advanced_packaging/models/stock_move.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_assign(self):
+        """Extend _action_assign to unreserve partial lines and trigger full package reservation"""
```

Comment:
What do we mean by `to unreserve partial lines` ?

Reply:
Looks like a copy/paste error, @caleb-shelton change this to just mention full package reservation in the doc-string. :)